### PR TITLE
Translated docs/api-reference in Marathi

### PR DESCRIPTION
**Pull request by Abdulhussainjarif**

### CODE REVIEW

File: docs/docs/api-reference.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ title: Gatsby API Reference
 overview: true
 ---
 
-Learn more about Gatsby API methods and options, common files, and customizations.
+गॅटस्बी एपीआय पद्धती आणि पर्यायांबद्दल अधिक जाणून घ्या, सामान्य फाईल्स, आणि सानुकूलने.
```
Comment (gamitics) [inline docs/docs/api-reference.md]: add adhikh mahite sathi after-> घ्या

**Review by gamitics — DISMISSED** [review]

**Comment by Abdulhussainjarif** [comment]
I have made changes as per the requested suggestions, please let me know if you need help with anything else.

Thanks and Regards, 
Abdulhussain Jarif

**Review by spielers — APPROVED** [review]
Hey @Abdulhussainjarif  Thanks for your Valuable Contributions 💐💐
we'll get you merged ASAP!

Best,
Spielers

**Comment by spielers** [comment]
Hey @Abdulhussainjarif 
Check out the https://www.gatsbyjs.com/contributing/contributor-swag/ for getting yourself a developer Swag🧑🏻‍💻

Best,
spielers

**Pull request merged**